backend/clickhouse/errors.go

```diff
@@ -648,7 +644,7 @@ func (client *Client) QueryErrorObjectsHistogram(ctx context.Context, projectId
 
 	orderBy := fmt.Sprintf("1 WITH FILL FROM %s(?, '%s') TO %s(?, '%s') STEP 1", aggFn, location.String(), aggFn, location.String())
 
-	sb, err := readErrorsObjects(selectCols, params, projectId, orderBy)
+	sb, err := readErrorsObjects(selectCols, params, projectId, "1", orderBy)
```

Comment:
My thought was this may be used for the error instances search in this card: https://linear.app/highlight/issue/HIG-4432/error-group-filters-should-apply-across-instances